readme.md

```diff
@@ -0,0 +1,153 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [🎯 Catalog of my things](#-catalog-of-my-things)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Video presentation](#video-presentation)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Test](#test)
+  - [👥 Authors ](#-authors-)
+    - [Authors](#authors)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [👋 Show your support ](#-show-your-support-)
+  - [🔭Acknowledgments ](#acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 🎯 Catalog of my things<a name="about-project"></a>
+
+> This is a console application that is built with ruby. It is used to manage items of an individual in a catalog. It has functionalities that include adding a book, music and game, with different properties attached to these items.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+  
+</details>
+
+<!-- prsentation vidéo -->
+
+### Video presentation
+
+[Video](https://drive.google.com/file/d/1GOK9NucJGQKEHaH4i_tv26YqYl6Upfmf/view?usp=sharing)
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add music album**
+- **Add books**
+- **Add games**
+

```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you explain your features in a professional way to make your readme more descriptive and professional 👍